python/tests/conftest.py

```diff
@@ -1,4 +1,53 @@
 import os.path
+import pytest
 import sys
+import yaml
+from typing import Dict, List, Union
 
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
\ No newline at end of file
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from looker.sdk import methods as mtds  # noqa: E402
+from looker.sdk import models as ml  # noqa: E402
+
+
+@pytest.fixture(scope="session")
+def client() -> mtds.LookerSDK:
+    client = mtds.LookerSDK.configure("../looker.ini")
+    return client
+
+
+@pytest.fixture(name="test_data")
+def get_test_data() -> Dict[str, Union[List[Dict[str, str]], str]]:
+    with open("../test/data.yml") as f:
+        test_data: Dict[str, Union[str, List[Dict[str, str]]]] = yaml.safe_load(f)
+
+    return {"users": test_data["users"], "email_domain": "@testfoo.com"}
+
+
+@pytest.fixture()
+def create_users(
+    client: mtds.LookerSDK, test_data: Dict[str, Union[List[Dict[str, str]], str]]
+) -> None:
+    # TODO: fix mypy type error on 'u'
+    # Create some users
+    users = test_data["users"]
```

Comment:
maybe something with https://mypy.readthedocs.io/en/latest/casts.html like

```python
users = cast(List[Dict[str, str]], test_data["users"])
```

I love this:

> give the type checker a little help when it can’t quite understand what is going on.

Reply:
This is great, nice find!